target/linux/mediatek/dts/mt7981b-zbtlink-zbt-z8103ax.dts

```diff
@@ -0,0 +1,262 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT 
+
+/dts-v1/;
+
+#include "mt7981.dtsi"
+
+/ {
+	model = "ZBT Z8103AX";
```

Comment:
Why not `Zbtlink ZBT-Z8103AX`? Probably the device model in the commit title ("[mediatek: add support for ZBT Z8103AX](https://github.com/openwrt/openwrt/pull/13933/commits/936eebdd11b6cdf10c68374d9b0b9e004c4059c7)") also should be fixed.

Reply:
I think ZBT Z8103AX will do for the 'model'. I based it on the existing routers made by Zbtlink.
https://github.com/openwrt/openwrt/commit/046263095e007ba80bdf3525de32d7cce11734e8

Reply:
Hmm... Indeed, we have 1 ZBT and 21 Zbtlink https://github.com/search?q=repo%3Aopenwrt%2Fopenwrt+model+%3D+"ZBT&type=code&p=1. Nevertheless, I suppose we need either ZBT or Zbtlink in one commit, not mix.